recommender/google/cloud/recommender_v1beta1/gapic/recommender_client.py

```diff
@@ -234,7 +234,7 @@ def list_recommendations(
     ):
         """
         Lists recommendations for a Cloud project. Requires the
-        recommender.\*.list IAM permission for the specified recommender.
+        recommender.*.list IAM permission for the specified recommender.
```

Comment:
This looks fine in the generated docs: 
![image](https://user-images.githubusercontent.com/8822365/73107244-30ad8380-3eb2-11ea-866a-1f2acc62b16c.png)